format/index.md

```diff
@@ -1304,11 +1304,11 @@ Accept: application/vnd.api+json
 
 #### Updating a Resource's Attributes <a href="#crud-updating-resource-attributes" id="crud-updating-resource-attributes" class="headerlink"></a>
 
-Any or all of a resource's attributes **MAY** be included in the resource
+Any or all of a resource's [attributes] **MAY** be included in the resource
 object included in a `PATCH` request.
 
-If a request does not include all of the [fields] for a resource, the server
-**MUST** interpret the missing [fields] as if they were included with their
+If a request does not include all of the attributes for a resource, the server
```

Comment:
Fields is the correct reference here. Relationships may be omitted in the same fashion as fields.